### DISCUSSION TITLE
Shlink and high availability

### DISCUSSION THREAD

**Discussion by fibis** [shlinkio/discussions#60] [Help wanted] [answered]
We have Shlink deployed in our Kubernetes cluster. To ensure high availability, we want to work with multiple replicas. Is this out of the box possible with  Shlink or will internal caching mechanisms have problems with this?

**Comment by acelaya** [comment, marked as answer]
It is supported, but you will need redis so that all the instances can share some information https://shlink.io/documentation/install-docker-image/#multi-instance-considerations
